server/conf/application.conf

```diff
@@ -307,6 +307,7 @@ play.filters {
   enabled += filters.HSTSFilter
   enabled += filters.LoggingFilter
   enabled += filters.ValidAccountFilter
```

Comment:
For posterity, it'd be great to include in the PR a before / after CSS size as well as a pointer to the docs for enabling this filter. That'll help future readers understand where we pulled this from.

Reply:
Disregard my request for a doc pointer since it was in fact already included in the PR details (whoops!).

Before / after CSS size would still be valuable in my opinion.